src/views/Dashboard.vue

```diff
@@ -9,13 +9,9 @@
 
     <v-container id="dashboard-container" class="view-container">
       <article id="dashboard-article">
-        <header>
-          <h1 data-test-id="dashboard-title">Dashboard</h1>
-        </header>
-
         <v-row>
           <v-col cols="12" md="9">
-            <section>
+            <section v-if="!isHistorical">
```

Comment:
Where did you see that the Todo List is hidden when the entity is historical?

Reply:
https://projects.invisionapp.com/share/XY11SF2SDP27#/screens/458580685

Reply:
![Screen Shot 2021-12-10 at 10 40 10 AM](https://user-images.githubusercontent.com/53542131/145625012-cd1d9bf6-bc67-46c0-9d5b-1ee16fc015c3.png)

Sorry, in the event the design is locked behind request access.

Reply:
Perfect, thanks.